### DISCUSSION TITLE
Wing with different airfoil section

### DISCUSSION THREAD

**Discussion by aweproject2023** [byuflowlab/FLOWUnsteady#66] [Q&A] [answered]
Hi,

I was running Wing example everything is fine. However, I wasn't sure how to apply different airfoil section in the script (wing.jl). Thx!

**Comment by aweproject2023** [comment]
How I can change the airfoil (RAE 101) profile used in the example to different airfoil shapes?

**Comment by EdoAlvarezR** [comment, marked as answer]
Hi Peter! 

The ["Simple Wing" tutorial](https://flow.byu.edu/FLOWUnsteady/examples/wing-4p2aoa/) shows how to pass the airfoil polar to the force calculation function, which is also shown in Line 170 here: [LINK](https://github.com/byuflowlab/FLOWUnsteady/blob/master/examples/wing/wing.jl#L170).

The file name of the airfoil polar refers to one of the airfoils in the default airfoil database that comes with FLOWUnsteady under [`database/airfoils`](https://github.com/byuflowlab/FLOWUnsteady/tree/master/database). See the slides under the [Propeller tutorial](https://flow.byu.edu/FLOWUnsteady/examples/propeller-J040/) for more details on the database. 

Alternatively, you can define your own database and pass the directory of the database to [`generate_calc_aerodynamicforce`](https://flow.byu.edu/FLOWUnsteady/api/flowunsteady-monitor/#FLOWUnsteady.generate_calc_aerodynamicforce) using the following keyword argument:

```julia
uns.generate_calc_aerodynamicforce(; add_parasiticdrag=true,
                                      add_skinfriction=true,
                                      airfoilpolar="myairfoilpolar.csv",
                                      parasiticdrag_args = (; read_path="path/to/my/database/airfoils")
                                      )
```

The polar file can be any polar downloaded from [airfoiltools.com](http://airfoiltools.com/), generated through [XFOIL](https://github.com/byuflowlab/Xfoil.jl), or that you have created yourself. For instance, here is what the RAE 101 polar file looks like: [LINK](https://github.com/byuflowlab/FLOWUnsteady/blob/master/database/airfoils/xf-rae101-il-1000000.csv)

**Reply by aweproject2023** [reply]
Awesome. Thank you so much!

**Comment by christianhauschel** [comment]
Am I right to assume that the polar cl values are not used in the calculation of the bound vorticity?
I.e. cambered airfoils would not be modelled accurately?

**Reply by EdoAlvarezR** [reply]
Correct. Cl of the polar is ignored in the wing model and only Cd is used.

**Reply by dlmmssm** [reply]
Hi, how can I calculate a wing with asymmetric airfoil? Pre-Twist the airfoil section by the zero lift angle?

> Correct. Cl of the polar is ignored in the wing model and only Cd is used.

**Reply by anuj117** [reply]
> Am I right to assume that the polar cl values are not used in the calculation of the bound vorticity? I.e. cambered airfoils would not be modelled accurately?

I have been trying to model an eVTOL design in Flow Unsteady to get mid fidelity aerodynamic estimates. However, I am having trouble changing the airfoil in the wing to a cambered one. If the software cannot model cambered airfoils, then it is not be suitable for such cases,right?
